account.go

```diff
@@ -117,14 +123,21 @@ func (c *Client) fetchBlob(ctx context.Context) ([]byte, error) {
 	return decodeBase64(blobBase64)
 }
 
+// ParseEncryptedAccounts parses encrypted accounts into a []*Account.
+// The original encrypted accounts data can be obtained from LastPass
+// using the FetchEncryptedAccounts method.
+func (c *Client) ParseEncryptedAccounts(r io.Reader) ([]*Account, error) {
+	return c.parseBlob(r)
+}
```

Comment:
Same here:
I pushed a commit where `parseBlob()` (renamed to `ParseEncryptedAccounts()`) is exported directly instead of adding another wrapper method.